SECURITY.md

```diff
@@ -1,3 +1,5 @@
 # Polkaswap Security Policy
 ## Reporting a Vulnerability
-If you think you have discovered a security issue in Polkaswap project, please let us know. We take all security bugs seriously and if confirmed upon investigation we will patch it as soon as possible and release a public security bulletin discussing the impact and crediting the discoverer. Reporting a security weakness is easy - just email us a description of the flaw and any related information (e.g. reproduction steps, version) at polkaswap@soramitsu.co.jp
+If you think you have discovered a security issue in the Polkaswap project, please let us know. 
+We take all security bugs seriously. If confirmed upon investigation, the issue will be addressed as soon as possible. We will release a public security bulletin discussing the impact and crediting the discoverer. 
+To report a security issue, send the description of the flaw and any related information (e.g. reproduction steps, version) to [polkaswap@soramitsu.co.jp](mailto:polkaswap@soramitsu.co.jp).
```

Comment:
maybe it's better to provide TG group for troubleshooting as well? What do you think @WRRicht3r ?

Reply:
Also, users can create issues directly using GH issues section
https://github.com/sora-xor/polkaswap-exchange-web/issues

We can add a template for bug reports, what do you think about it? @WRRicht3r

Reply:
Hey @stefashkaa I don't know how to add the templates, so it would be much appreciated, thank you.